build.sbt

```diff
@@ -2248,7 +2248,8 @@ lazy val `microsite` = project.in(file("doc/microsite"))
       "org.http4s" %% "http4s-dsl" % V.http4s,
       "org.http4s" %% "http4s-circe" % V.http4s,
       "org.http4s" %% "http4s-blaze-server" % V.http4s,
-      "org.http4s" %% "http4s-blaze-client" % V.http4s
+      "org.http4s" %% "http4s-blaze-client" % V.http4s,
+      "org.tpolecat" %% "skunk-core" % V.skunk,
```

Comment:
This will not persist after build regeneration with `./sbtgen.sc`, dependencies have to be added in `project/Deps.sc`